modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -760,4 +760,25 @@ definition {
 			locator1 = "PRMMDFRequest#STATUS_BUTTON");
 	}
 
+	@summary = "Assert the value for Budget MDF Request"
+	macro viewValueBudget{
+		if (isSet(totalCost)) {
+			AssertTextEquals(
+				key_fieldName = "Total cost",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
+
+		if (isSet(claimPercent)) {
+			AssertTextEquals(
+				key_fieldName = "Claim Percent",
+				key_fieldValue = ${fieldValue},
+				locator1 = "PRMMDFRequest#VALUE_BUDGET");
+		}
```

Comment:
Hi @Italo-gabriel, the `claimPercent` will always be 50%, so this assert isn't necessary

Reply:
Also, remove the `isSet` conditions, since your macro will assert just the Total cost value.

Reply:
Done !